Add `jsondata:Name` and `http:Header` mapping for record fields in the OpenAPI generation (#1773)
ballerina-platform/openapi-tools: docs/ballerina-to-oas/spec/spec.md

</tr>
</table>

> > **Note:** If the record type has `jsondata:Name` annotation, then the name specified in the annotation is used as the schema name.

Reviewer: Should this be
```suggestion
> **Note:** If any field in the record type has a `jsondata:Name` annotation, the name specified in the annotation will be used as the schema name.
```
?

> 
> Ballerina record type:
> ```ballerina
> public type Album record {|
>     string artist;
>     @jsondata:Name {value: "_id"}
>     string id;
>     string title;
> |};
> ```
> 
> Generated schema:
> ```yml
> Album:
>   required:
>   - id
>   - artist
>   - title
>   type: object
>   properties:

> >     id:

Author: Fixed via [7b0c2f7](https://github.com/ballerina-platform/openapi-tools/pull/1773/commits/7b0c2f7d8f2b6bee944d99fdb8dd45456bac8870)
